Update umb property (#5712)
umbraco/UmbracoDocs: 10/umbraco-cms/reference/angular/directives/umbproperty.md

</umb-property>
```

> `Properties` contains the model for each property. We use `ng-repeat` to iterate over each property, passing them to the two directives via `property` and `model` attributes.

Reviewer: ```suggestion
`Properties` contains the model for each property. `ng-repeat` can be used to iterate over each property, passing them to the two directives via `property` and `model` attributes.
```
